crates/config/src/madara.rs

```diff
@@ -155,20 +118,99 @@ impl MadaraRunnerConfigSequencer {
             };
 
             MadaraPreset { preset_type, path }
-        });
+        };
+
+        Ok(MadaraRunnerConfigSequencer { base_path, preset })
+    }
+}
+
+impl MadaraRunnerConfigDevnet {
+    /// Fill Devnet configuration using interactive prompts
+    pub fn fill_values_with_prompt() -> anyhow::Result<MadaraRunnerConfigDevnet> {
+        let base_path = Prompt::new("Input DB path:")
+            .default("./madara-devnet-db")
+            .ask();
+
+        Ok(MadaraRunnerConfigDevnet { base_path })
+    }
+}
+
+impl MadaraRunnerConfigMode {
+    /// Retrieve MadaraMode from current configuration
+    pub fn mode(&self) -> MadaraMode {
+        match &self.params {
+            MadaraRunnerParams::Devnet(_) => MadaraMode::Devnet,
+            MadaraRunnerParams::Sequencer(_) => MadaraMode::Sequencer,
+            MadaraRunnerParams::FullNode(_) => MadaraMode::FullNode,
+        }
+    }
+
+    /// Fill MadaraRunnerConfigMode with prompts based on selected mode
+    pub fn fill_values_with_prompt() -> anyhow::Result<MadaraRunnerConfigMode> {
+        let name = "Madara".to_string();
+        let mode: MadaraMode = PromptSelect::new("Select Madara mode:", MadaraMode::iter()).ask();
```

Comment:
@HermanObst:
There is an interactive way to select the Mode. If the user doesn't add any command/param, then the CLI will be interactive